index.html

```diff
@@ -12,46 +12,52 @@
   <link rel="stylesheet" href="style.css" >
 </head>
 <body>
+<a href="#"><img id="logo" src="https://img2.freepng.ru/20180423/wlw/kisspng-soldier-computer-icons-military-salute-5ade61d71d3dc2.8402920815245234791198.jpg"></a>
```

Comment:
pls download image to u local folder, for intance "images" and add to solution, because external resources can be slow down u page and is a potencial security risk

Reply:
got it